lib/dal-materialized-views/src/component_list.rs

```diff
@@ -19,9 +19,9 @@ pub async fn assemble(ctx: DalContext) -> super::Result<ComponentListMv> {
     for component_id in component_ids {
         components.push(super::component::assemble(ctx.clone(), component_id).await?);
     }
-
+    let id: dal::WorkspacePk = ctx.workspace_pk()?;
```

Comment:
Would be nice to make the formatting of how this is added to the MVs consistent with each other.